frontend/test/metabase-visual/dashboard/filter-sidebar.cy.spec.js

```diff
@@ -0,0 +1,33 @@
+import { restore } from "__support__/e2e/cypress";
+
+const questionDetails = {
+  query: { "source-table": 2 },
```

Comment:
Let's import `SAMPLE_DATASET` and get `ORDERS_ID` from there instead of hard coding it, please.
```suggestion
  query: { "source-table": ORDERS_ID },
```

Reply:
👍 Done